tests/saw/Makefile

```diff
@@ -254,7 +254,7 @@ s2n/pq-crypto/sike_r1 :
 s2n/pq-crypto/sike_r2 :
 	mkdir -p $@
 
-s2n/pq-crypto/bike :
+s2n/pq-crypto/bike_r1 :
```

Comment:
What you have here probably builds, but there are a few other places in `tests/saw/Makefile` that should be renamed bike -> bike_r1, otherwise there will be conflicts in the future with bike_r2. Checkout out https://github.com/awslabs/s2n/commit/1e6c8f457e0029bc087aab842748ba34b46d5f62 for where the sike changes were made.

Reply:
Done